src/update/UpdateManager.cpp

```diff
@@ -130,6 +130,42 @@ void UpdateManager::GetUpdates(bool ignoreNetworkErrors)
 
                 if (code == wxID_YES)
                 {
+                    const audacity::network_manager::Request downloadRequest(mVersionPatch.download.ToStdString());
+                    auto downloadResponse = audacity::network_manager::NetworkManager::GetInstance().doGet(downloadRequest);
+
+                    downloadResponse->setRequestFinishedCallback([downloadResponse, this](audacity::network_manager::IResponse*) {
+                        mProgressDialog.reset();
+                        }
+                    );
+
+                    downloadResponse->setDownloadProgressCallback(
+                        [downloadResponse, this](int64_t current, int64_t expected) {
+
+                            static std::once_flag flag;
+                            std::call_once(flag, [this] {
+                                const auto title = XO("Audacity update")
+                                    .Translation();
+
+                                auto audacityPatchFilename = wxFileName(mVersionPatch.download).GetName();
+                                const auto message = XO("Downloading %s")
+                                    .Format(audacityPatchFilename)
+                                    .Translation();
+
+                                mProgressDialog.reset(new wxGenericProgressDialog(title, message,
+                                    1000000,    // range
+                                    nullptr,    // parent
+                                    wxPD_APP_MODAL | wxPD_ELAPSED_TIME | wxPD_SMOOTH));
+
+                                auto audacityInstallerPath = wxFileName(
```

Comment:
This variable is not yet used.  I suppose a later commit?  Then it might be added with that commit.